sdks/python/apache_beam/transforms/util.py

```diff
@@ -273,7 +279,7 @@ def div_keys(kv1_kv2):
     pairs = sorted(zip(sorted_data[::2], sorted_data[1::2]),
                    key=div_keys)
     # Keep the top 1/3 most different pairs, average the top 2/3 most similar.
-    threshold = 2 * len(pairs) / 3
+    threshold = 2 * len(pairs) // 3
```

Comment:
Let's use past.utils.division.old_div in line 280 as an exception, and add a TODO(BEAM-4858) comment to clean this up.

Reply:
I have confirmed that this change brings performance back to the same ballpark.

Reply:
Perfect! I will update the PR, thx!

Reply:
Thanks for your patience with this investigation.